src/main/webapp/src/app/find-gardens-page/find-gardens.component.css

```diff
@@ -0,0 +1,46 @@
+.avatar {
+  background-size: cover;
+  height: 50px;
+  padding-top: 5px;
+}
+.avatar-image {
+  width: 50px;
+}
+
+.button {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0.5em 3em;
+  text-decoration: none;
+  border: none;
+  box-sizing: border-box;
+  box-shadow: inset 0 -0.6em 0 -0.35em #003353;
+  background-color: #005c80;
+  color: white;
+}
+.cards {
+  max-width: 400px;
+  margin-left: auto;
```

Comment:
You can say this more concisely via `margin: 20px auto`;

Reply:
Done!